.github/workflows/test.yml

```
@@ -59,6 +59,5 @@ jobs:
 
       - name: Tox tests
         shell: bash
-        # Drop the dot: py3.7 -> py37
         run: |
-          tox -e py`echo ${{ matrix.python-version }} | tr -d .`
+          tox -e py
```

does this work the same?

```suggestion
          tox
```

---

no it doesn't